samza-api/src/main/java/org/apache/samza/operators/windows/Windows.java

```diff
@@ -84,6 +87,8 @@
  * and triggers are fired and window panes are emitted per-key. It is possible to construct "keyed" variants of all the above window
  * types.
  *
+ * <p> Time granularity for windows: Currently, time durations are always measured in milliseconds. Time units of
```

Comment:
Thanks for moving. This still sounds like a vague disclaimer - not sure if it's helpful for users. If we know what we specifically don't support, would be good to document that (maybe as an example).